examples/language/llama/benchmark.py

```diff
@@ -5,6 +5,8 @@
 from contextlib import nullcontext
 
 import torch
+
+torch.autograd.set_detect_anomaly(True)
```

Comment:
remove debug info

Reply:
> remove debug info

Fixed in https://github.com/hpcaitech/ColossalAI/pull/6083/commits/e76308c6e65cb73cc5b20936bd232ba7390c6b11